docs/source/getting_started/quick_start.rst

```diff
@@ -145,24 +146,21 @@ e. Set Up Individual Modules
 
             .. note :: The `*.pdb`/`*.spf` files above do not yet exist; you specify the file paths where `MolProp` module will generate them.
 
-            .. note :: This step is the same for both production and test runs.
-
 
         .. figure:: quick_start/quick_start_Deposit_mols.png
            :alt: deposit_molecules_input
            :width: 100%
            :align: center
 
 
-
-
     3. **ESAnalysis**
 
-
         * In the ``General`` tab of the ESAnalysis module, adapt the following:
-
             * `Morphology`: `Deposit3/outputs/structurePBC.cml` (again using the rightmost button)
-            * For a quick test, disable computation of absolute values and compute disorder and couplings only for a small shell
+
+        * For **Test Runs** only or if the absolute energy levels is not important:
```

Comment:
Test runs are not meant to deliver any meaningful results, so how could an absolute level be important? I don't have a strong opinion about this, but in any case it should be indented to one lower level, as it is a subbullet of General tab

Reply:
Test Runs OR Production runs where absolute values are not important

Reply:
this is the meaning of that. test runs _or_ <...>

Reply:
Ok, got it. Fine by me, but the indent should be fixed.

Reply:
not sure what you mean under "indent", I made some change so that the thing is not bold there, see nothing wrong with indents

Reply:
The new bullet is on the same level als the first bullet with "General" tab and a bit farther below "Engines" tab. It should be a sub bullet of Generalt bullet